shell/platform/darwin/ios/framework/Source/platform_views_controller.mm

```diff
@@ -2,22 +2,18 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#import "flutter/shell/platform/darwin/ios/framework/Source/FlutterPlatformViews_Internal.h"
-
-#include <Metal/Metal.h>
-#include <unordered_map>
+#import "shell/platform/darwin/ios/framework/Source/platform_views_controller.h"
 
 #include "flow/surface_frame.h"
 #include "flutter/flow/view_slicer.h"
 #include "flutter/fml/make_copyable.h"
-#include "flutter/fml/platform/darwin/scoped_nsobject.h"
+#include "fml/synchronization/count_down_latch.h"
+
 #import "flutter/shell/platform/darwin/ios/framework/Source/FlutterOverlayView.h"
 #import "flutter/shell/platform/darwin/ios/framework/Source/FlutterView.h"
 #import "flutter/shell/platform/darwin/ios/ios_surface.h"
-#include "fml/logging.h"
-#include "fml/synchronization/count_down_latch.h"
 
-FLUTTER_ASSERT_ARC
```

Comment:
Should this file be renamed `.cc` if there's no more Obj-C in it?

Reply:
We're still doing objective C stuff in here

Reply:
We should leave this in, in that case, to blow up if we compile with ARC disabled.